app/PrintAccount.php

```diff
@@ -11,6 +11,11 @@ class PrintAccount extends Model
     public $incrementing = false;
     public $timestamps = false;
 
+    const COST = [
+        'twosided' => 12,
+        'onesided' => 8,
```

Comment:
We should not leave this in the source. Maybe .env is enough?

Reply:
Sounds reasonable to me too.

Reply:
Done.